app/controllers/api/users.php

```diff
@@ -71,7 +71,7 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
             : ID::custom($userId);
 
         if ($project->getAttribute('auths', [])['personalDataCheck'] ?? false) {
-            $personalDataValidator = new PersonalData($userId, $email, $name, $phone);
+            $personalDataValidator = new PersonalData($userId, $email, $name, $phone, false, true);
```

Comment:
Let's use named parameters so it's clear what false and true are doing without checking the class